server/clientFactory.js

```diff
@@ -50,11 +50,13 @@ function clientFactory(config) {
     Network = HSNetwork;
     NodeClient = HSNodeClient;
     WalletClient = HSWalletClient;
-  } else {
+  } else if (chain === 'bitcoin' || chain === 'bitcoincash') {
```

Comment:
This is ok for now, but may be more maintainable as a `switch` over time

Reply:
Good call. I'll update as it looks cleaner anyway

Reply:
Updated to switch 👍